applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -7,12 +7,7 @@
 import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
 
 # Importing the base class
-from python_solver import PythonSolver
-
-
-def CreateSolver(model, custom_settings):
```

Comment:
@armingeiser I am only removing from the base class
It is not needed here / doesn't make sense here